REST_API/README.md

```diff
@@ -0,0 +1,38 @@
+# Call Kobiton REST API to get session information
+
+This repository includes examples to demonstrate how to get Kobiton session information and session commands with NodeJS.
+> Note: Kobiton also provides other languages to fetch Kobiton Rest Api, visit [here](https://api.kobiton.com/docs) for more information.
+## Prerequisites
+- Kobiton account
+- Install node and npm
+## Usage
++  Clone this repository
++  Replace your `KOBITON_USERNAME`, `KOBITON_API_KEY`, `SESSION_ID` to the sample script
+
+```
+const username='<YOUR_USERNAME>';
+const apikey='<YOUR_APIKEY>';
+const session = '<SESSION_ID>';
+```
+
++ Run this command line: `npm install` to setup dependencies
+> Note: you just only need to execute 'npm install' for the first running time.
+## Get sessions information
```

Comment:
```
## Get session data
- Execute:
        `node get-session-info.js`

- Request command: 
        `GET /sessions/{sessionId}`
Retrieve a session info belonging to the current user or organization. 
> Note: Absolute urls in response body like ‘screenshotDownloadUrl’, ‘previewUrl’, etc. expire in 2 hours since the response is made

For more information, please visit https://api.kobiton.com/docs/#get-sessions
```